From d0d2c4c98fa7a978459da04060fe68ec96b84773 Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Fri, 29 Feb 2008 23:13:09 +0200
Subject: [PATCH] Makefile: Fix install target

Signed-off-by: Jari Aalto <jari.aalto@cante.net>
---
 Makefile |   12 ++++++------
 1 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/Makefile b/Makefile
index 6b3324f..8ef5dcb 100644
--- a/Makefile
+++ b/Makefile
@@ -6,10 +6,10 @@ INSTALL ?= install
 
 #---------------- Do not modify below this point ------------------
 
-INSTALL_DIR     := $(INSTALL) -p -d -o root -g root -m 0755
-INSTALL_FILE    := $(INSTALL) -p    -o root -g root -m 0644
-INSTALL_PROGRAM := $(INSTALL) -p    -o root -g root -m 0755 # -s
-INSTALL_SCRIPT  := $(INSTALL) -p    -o root -g root -m 0755
+INSTALL_DIR     := $(INSTALL) -p -d  -m 0755
+INSTALL_FILE    := $(INSTALL) -p     -m 0644
+INSTALL_PROGRAM := $(INSTALL) -p     -m 0755 # -s
+INSTALL_SCRIPT  := $(INSTALL) -p     -m 0755
 
 VERSION = $(shell cat VERSION)
 CFLAGS += -DVERSION=\"$(VERSION)\"
@@ -56,9 +56,9 @@ distclean: clean
 
 install: all
 	$(INSTALL_DIR) $(DESTDIR)$(PREFIX)/bin
-	$(INSTALL_PROGRAM) readpst{,log} $(DESTDIR)$(PREFIX)/bin
+	$(INSTALL_PROGRAM) readpst.exe readpstlog.exe $(DESTDIR)$(PREFIX)/bin
 	$(INSTALL_DIR) $(DESTDIR)$(PREFIX)/share/man/man1
-	$(INSTALL_FILE) readpst{,log}.1 $(DESTDIR)$(PREFIX)/share/man/man1/
+	$(INSTALL_FILE) *.1 $(DESTDIR)$(PREFIX)/share/man/man1/
 	$(INSTALL_DIR) $(DESTDIR)$(PREFIX)/share/doc/libpst
 	$(INSTALL_FILE) $(DOCS) $(DESTDIR)$(PREFIX)/share/doc/libpst/
 
-- 
1.5.3.8

