From: Jari Aalto <jari.aalto@cante.net>
Subject: In x86-32 the linking needs Python libraries

---
 python/Makefile.in |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/python/Makefile.in
+++ b/python/Makefile.in
@@ -143,7 +143,7 @@ LIBICONV = @LIBICONV@
 LIBOBJS = @LIBOBJS@
 LIBPST_SO_MAJOR = @LIBPST_SO_MAJOR@
 LIBPST_VERSION_INFO = @LIBPST_VERSION_INFO@
-LIBS = @LIBS@
+LIBS = @LIBS@ -lpython2.7
 LIBTOOL = @LIBTOOL@
 LIPO = @LIPO@
 LN_S = @LN_S@
