From 34144a17ec7bb89cb7445a12be26c876bbb739cc Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Fri, 29 Feb 2008 19:10:51 +0200
Subject: [PATCH] readpst.c: change int32_t to int

Signed-off-by: Jari Aalto <jari.aalto@cante.net>
---
 readpst.c |   21 ++++++++++++++-------
 1 files changed, 14 insertions(+), 7 deletions(-)

diff --git a/readpst.c b/readpst.c
index 5b4cfaa..67a4071 100644
--- a/readpst.c
+++ b/readpst.c
@@ -69,19 +69,26 @@ struct file_ll {
 // Function Declarations {{{1
 void  write_email_body(FILE *f, char *body);
 char *removeCR (char *c);
-int32_t   usage();
-int32_t   version();
+//int32_t   usage();
+int   usage();
+//int32_t   version();
+int   version();
 char *mk_kmail_dir(char*);
-int32_t   close_kmail_dir();
+//int32_t   close_kmail_dir();
+int   close_kmail_dir();
 char *mk_recurse_dir(char*);
-int32_t   close_recurse_dir();
+//int32_t   close_recurse_dir();
+int   close_recurse_dir();
 char *mk_seperate_dir(char *dir, int overwrite);
-int32_t   close_seperate_dir();
-int32_t   mk_seperate_file(struct file_ll *f);
+//int32_t   close_seperate_dir();
+//int32_t   mk_seperate_file(struct file_ll *f);
+int   close_seperate_dir();
+int   mk_seperate_file(struct file_ll *f);
 char *my_stristr(char *haystack, char *needle);
 char *check_filename(char *fname);
 char *rfc2426_escape(char *str);
-int32_t chr_count(char *str, char x);
+//int32_t chr_count(char *str, char x);
+int chr_count(char *str, char x);
 char *rfc2425_datetime_format(FILETIME *ft);
 char *rfc2445_datetime_format(FILETIME *ft);
 char *skip_header_prologue(char *headers);
-- 
1.5.3.8

